daemon/api.go

```diff
@@ -62,36 +62,43 @@ var api = []*Command{
 
 var (
 	rootCmd = &Command{
-		Path: "/",
-		GET:  SyncResponse([]string{"/1.0"}).Self,
+		Path:    "/",
+		GuestOK: true,
+		GET:     SyncResponse([]string{"/1.0"}).Self,
 	}
 
 	v1Cmd = &Command{
-		Path: "/1.0",
-		GET:  v1Get,
+		Path:    "/1.0",
+		GuestOK: true,
+		GET:     v1Get,
 	}
 
+	// XXX: this and the following could arguably be information leaks
```

Comment:
I was thinking the same about the above. I can't think of a good reason for us to serve anything for completely anonymous users. Is there a use case?